fossunited/www/newsletter/index.html

```diff
@@ -0,0 +1,366 @@
+{% extends "templates/foss_base.html" %}
+{% block page_content %}
+<section id="header" class="h-100">
+  <div class="container">
+    <div id="hero" class="mt-5 d-flex flex-column justify-content-center ">
+      <div class="news-icon d-flex justify-content-center">
+        <img height="150px" width="150px" src="/assets/fossunited/icons/newsletter.svg" />
+      </div>
+      <div id="hero" class="text-center">
+        <h1 class="press2p">
+          The FOSS United Newsletter
+        </h1>
+        <p>
+        Monthly digest of the latest news and updates about FOSS grants, events and mon school. Subscribe to stay updated and get news from us directly in your inbox.
+        </p>
+      </div>
+    </div>
+    <div id="subscribe-section">
+      <div class="container-center my-5 d-flex flex-column flex-md-row flex-lg-row">
+        <div class="mr-2 py-3 ">
+          <label for="email"></label>
+          <input id="email" type="email" placeholder="johndoe@example.com" name="email_address" />
+        </div>
+        <div>
+          <button id="subscribe">
+            Subscribe
+          </button>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+<section id="newsletters-list">
+  <div class="base-container container" style="max-width: 1000px">
+    <h4 class="text-center">Past Newsletters</h4>
+    <div>
+      <ul class="m-auto py-8 list-unstyled gap-2 text-left row justify-content-center">
+        {% for newsletter in newsletters %}
+        <li id="news" class="py-4 text-center col-md-3 position-relative">
+          {{ newsletter.subject }}
+          <a class="stretched-link" href="{{ frappe.get_url() }}/{{ newsletter.route }}"></a>
+        </li>
+        {% endfor %}
```

Comment:
Kind of redundant. I suggest replacing the `<li>` with `<a>` and remove line 41.